website/snippets/_v2-sl-prerequisites.md

```diff
@@ -7,4 +7,4 @@
 - Set up the [Semantic Layer API](/docs/dbt-cloud-apis/sl-api-overview) in the integrated tool to import metric definitions. 
   - dbt Core or Developer accounts can define metrics but won't be able to dynamically query them.<br />
 - Understand [MetricFlow's](/docs/build/about-metricflow) key concepts, which powers the latest dbt Semantic Layer.  
-- Note that SSH tunneling for [Postgres and Redshift](/docs/cloud/connect-data-platform/connect-redshift-postgresql-alloydb) connections, [PrivateLink](/docs/cloud/secure/about-privatelink), and [Single sign-on (SSO)](/docs/cloud/manage-access/sso-overview) doesn't support the dbt Semantic Layer yet.
+- Note that SSH tunneling for [Postgres and Redshift](/docs/cloud/connect-data-platform/connect-redshift-postgresql-alloydb) connections, and [PrivateLink](/docs/cloud/secure/about-privatelink) doesn't support the dbt Semantic Layer yet.
```

Comment:
I feel like this should be:
```
Note that the dbt Semantic Layer doesn't yet support SSH tunneling for [Postgres and Redshift](/docs/cloud/connect-data-platform/connect-redshift-postgresql-alloydb) connections, and [PrivateLink](/docs/cloud/secure/about-privatelink).
```

Reply:
great feedback, thank you @aiguofer ! folded it in now 🙏